processing/src/main/java/io/druid/segment/data/CachingIndexed.java

```diff
@@ -108,7 +109,14 @@ public void close() throws IOException
     }
   }
 
-private static class SizedLRUMap<K, V> extends LinkedHashMap<K, Pair<Integer, V>>
+  @Override
+  public void inspectRuntimeShape(RuntimeShapeInspector inspector)
+  {
+    inspector.visit("cachedValues", cachedValues != null);
```

Comment:
Because `CachingIndexed.get()` doesn't call methods on instances of this type directly. So for Hotspot, it's just an object, doesn't matter of what type. If it does matter in `delegate.get()`, `delegate`'s runtime shape should reflect this, delegate is inspected in `CachingIndexed.inspectRuntimeShape()`.